### ISSUE TITLE
Nvda does not read the name of the object to be selained when there is only one object in the google worksheet

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7938]
### Steps to reproduce:
1. Open a new spreadsheet in google docs.
1. Enable screenreader support with ctl+alt+z
1. Inside this worksheet insert an object, it can be an image.
1. Now press alt + shift + a to open the google worksheets accessibility menu.
1. Use the arrow keys to go to select, NVDA reports `Select e ►  subMenu  1 of 1`
1. Press the keyboard right arrow to open this submenu. Notice that the name of this item is not spoken only `menu  subMenu`

### Expected behavior:
The nvda should read the name of the object that can be selected. Eg `Image: Image 1 of 1`

### Actual behavior:
Nvda does not read the name of the object that can be selected. Only `menu submenu`
If you press enter even the nvda not reading the name of the object it will be selected and opened in the worksheet.
If there is more than one object assigned to the worksheet, nvda reads the name of all objects in the menu as expected.
In other submenus the first item is not announced, the focus must be moved down and back up.

### System configuration:
NVDA version:
> Example: next-14799

NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 7

Name and version of other software in use when reproducing the issue:
firefox 57
### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result

**Comment by feerrenrut** [comment]
I can reproduce this, I have edited the description to make it a bit clearer. The DevInfo for the selected item does not have the name "Image: Image", this could be a timing bug, or a missing focus bug. It may be best to raise a support request with Google Docs suite.

**Comment by Adriani90** [comment]
@fernando-jose-silva can you try to reproduce this with NVDA 2020.1? I cannot reproduce it is seems this entry is not in the accessibility menu anymore.

**Comment by fernando-jose-silva** [comment]
I also can't reproduce the problem anymore, the report item is no longer present in accessibility.

**Comment by ehollig** [comment]
Closing as worksforme as mentioned in https://github.com/nvaccess/nvda/issues/7938#issuecomment-629847573
